Back-End/src/main/java/com/flight_search/mapper/SegmentMapper.java

```diff
@@ -0,0 +1,14 @@
+package com.flight_search.mapper;
+
+import com.flight_search.dto.SegmentDTO;
+import com.flight_search.model.Flight.Segment;
+import org.mapstruct.Mapper;
+import org.mapstruct.factory.Mappers;
+
+@Mapper
+public interface SegmentMapper {
+    SegmentMapper INSTANCE = Mappers.getMapper(SegmentMapper.class);
```

Comment:
same here why INSTANCE?

also is INSTANCE used?

Reply:
Stop using mappers and instead only use DTOs to parse the flight information.